src/components/ChatLog.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+
+const ChatLog =  ({ entries, handleLike }) => {
+
+    const chatComponents = entries.map((message, index) => {
+
+        return (
+            <li key={index}>
```

Comment:
Conventionally, we will use unique identifiers like `ids` of the objects that are passed down. There's nothing wrong with this! Just wanted to keep you in the loop, just in case you start to notice industry!